policyengine_us/parameters/gov/states/id/tax/income/subtractions/subtractions.yaml

```
@@ -0,0 +1,15 @@
+description: Idaho subtracts these items when calculating state adjusted gross income.
+values:
+  2021-01-01:
+    - cdcc_relevant_expenses
+    - us_govt_interest
```

Similarly reference subsection for other items.
```suggestion
    - us_govt_interest # (3)(a)
```